site/content/docs/main/restore-hooks.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Restore Hooks"
+layout: docs
+---
+
+Velero supports Restore Hooks, custom actions that can be executed during or after the restore process. There are two kinds of Restore Hooks:
+
+1. InitContainerRestoreHooks: These will add init containers into restored pods to perform any necessary setup before the application containers of the restored pod can start.
```

Comment:
I don't think we've agreed on what to call these.
I am going to suggest a third variant `InitContainer RestoreHooks`.

Reply:
@a-mccarthy Thanks for bringing up that point. Other than the internal variables I think Ashish is right, we haven't landed on a name. Scanning [the design doc](https://github.com/vmware-tanzu/velero/blob/main/design/restore-hooks.md), I don't see a term either.

I like `InitContainer Restore Hooks`. This is because we call the hooks on the backup side `Backup Hooks`, not `BackupHooks`.

@carlisia, @zubron, @areed Do any of you have a preference?

Reply:
I also prefer `InitContainer Restore Hooks`.

Reply:
updated the docs accordingly. PTAL